code/processes/wdb.q

```diff
@@ -527,6 +527,6 @@ upd:.wdb.replayupd;
 / - clear any wdb data in the current partition
 .wdb.clearwdbdata[];
 /- initialise the wdb process
-.wdb.startup[];
+.wdb.startup[`];
```

Comment:
is this change necessary? I don't think this argument will be used, so don't think this should be needed?

Reply:
Ah, that was my fault, I didn't realise it would run without it so I suggested it while one a call with David (just did a mini test and I now understand :) )